shared/bundle_analysis/comparison.py

```diff
@@ -44,6 +44,7 @@ class ChangeType(Enum):
     bundle_name: str
     change_type: ChangeType
     size_delta: int
+    percentage_delta: float  # [-100, 100] range
```

Comment:
I don’t see any code that is explicitly clamping this to the said range. does that happen someplace thats not visible in this diff?

Reply:
Hum good point, not really. I'll remove the comment, as it would not be possible to clamp it and still have accurate numbers I guess